lib/VolumeManager/VolMgrPlatformSupportLinux.rb

```diff
@@ -40,25 +40,27 @@ def preMount
     desc = @ost.snapshotDescription ? @ost.snapshotDescription : "EVM Snapshot"
     st = Time.now
     @snMor = @vimVm.createEvmSnapshot(desc, "false", true, @ost.snapshot_create_free_space)
-    $log.info "VolMgrPlatformSupportLinux.preMount: VM snapshot created in [#{Time.now - st}] seconds"
+    $log.info "VolMgrPlatformSupportLinux.preMount: VM snapshot [#{@snMor}] created in [#{Time.now - st}] seconds"
     $log.debug "VolMgrPlatformSupportLinux.preMount: snMor = \"#{@snMor}\""
   end
 
   def postMount
-    $log.debug "VolMgrPlatformSupportLinux.postMount Enter: force = #{@ost.force}, @vimVm.nil? = #{@vimVm.nil?}"
+    log_prefix = "VolMgrPlatformSupportLinux.postMount"
+    $log.debug "#{log_prefix} Enter: force = #{@ost.force}, @vimVm.nil? = #{@vimVm.nil?}"
     return unless @ost.force
     return unless @vimVm
 
     if @ost.force
       if !@snMor
-        $log.warn "VolMgrPlatformSupportLinux.postMount: VM not snapped: #{@cfgFile}"
+        $log.warn "#{log_prefix}: VM not snapped: #{@cfgFile}"
       else
-        $log.debug "VolMgrPlatformSupportLinux.postMount: removing snapshot snMor = \"#{@snMor}\""
+        $log.debug "#{log_prefix}: removing snapshot snMor = \"#{@snMor}\""
```

Comment:
Make this at least info to coincide with the call in preMount.

Reply:
I was thinking the pre/post symmetry was that they both logged to info the result after the operation:
`$log.info "VolMgrPlatformSupportLinux.preMount: VM snapshot [#{@snMor}] created in [#{Time.now - st}] seconds"`
and
`$log.info "#{log_prefix}: VM snapshot [#{@snMor}] removed"`
But I can just make this info.

Reply:
Or we can add a `INFO -- Creating snapshot` to preMount, thoughts?

Reply:
Oh you're right @agrare I didn't see the second line.